app/AbstractRequest.php

```diff
@@ -454,10 +454,7 @@ public function getLastLogContents(): string
      */
     public function getLastLogFile(): string
     {
-        if (!$this->sentLogs) {
-            throw new DomainException('No log files have been saved by this instance.');
-        }
-        return Str::finish($this->getLogFolder(), '/') . Arr::last($this->sentLogs);
+        return Arr::last($this->sentLogs) ?? throw new DomainException('No log files have been saved by this instance.');
```

Comment:
As written, logs will just fail for cache hits for a day while the cache turns over, because the cache won't have a folder and this assumes they do.

Couple ideas:
1. Fuck it, clear cache if we have to, selectively, by tags.
2. Add a shim in `responseFromCache` implement a shim where if logFiles has strings  with no `/` we insert this request instance's folder? (this isn't worse than what's in prod right now)
3. Update CACHE_KEY_SEED because the data stored is (barely) changing. This has the effect of immediately invalidating all cache, even if the vast majority of cache hits don't care about the log folder.

Reply:
Option 2 seems helpful if we think we need to easily get logs today. Otherwise number 1 is good with me.